lib/hanami/routing/http_router.rb

```diff
@@ -143,7 +143,7 @@ def options(path, options = {}, &blk)
       # @since 0.1.1
       # @api private
       def mount(app, options)
-        add("#{ options.fetch(:at) }*").to(
+        add("#{ options.fetch(:at) }*", host: options[:host]).to(
```

Comment:
@nicolaracco Using `:host` option both as `Router#initialize` and as a `#mount` option can be confusing. What about `:subdomain` instead?